matlab/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jupyter-matlab-proxy
+numpy
```

Comment:
why do you need numpy for matlab? in any case, these should be a part of the base image, see above.

Reply:
are these still needed? Presumably they are part of the base image?